fraqbot/tests/test_shakespeare.py

```diff
@@ -0,0 +1,39 @@
+import os
+import sys
+import threading
+
+from Legobot.Lego import Lego
+from mock import patch
+
+LOCAL_PATH = os.path.join(
+    os.path.abspath(os.path.dirname(__file__)),
+    '..',
+    'Local'
+)
+sys.path.append(LOCAL_PATH)
+from shakespeare import Shakespeare  # noqa: E402
+
+URL_BASE = 'http://localhost/files/'
+LOCK = threading.Lock()
+BASEPLATE = Lego.start(None, LOCK)
+LEGO = Shakespeare(BASEPLATE, LOCK, url_base=URL_BASE)
+
+
+def test_listening_for():
+    assert LEGO.listening_for({'text': 'shake'}) is False
+    assert LEGO.listening_for({'text': '!shake moin'}) is True
+    assert LEGO.listening_for({'text': '!shake'}) is True
+
+
+@patch('Legobot.Lego.Lego.reply')
+@patch('shakespeare.Shakespeare._get_quote')
```

Comment:
Do we need to mock this method? Since you're not testing what it returns this may not be necessary, unless you're concerned it's going to throw an exception.